services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from db.models import CinemaHall
+from django.db.models import QuerySet
+
+
+def get_cinema_halls() -> QuerySet:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(

```

Comment:
When we create CinemaHall, we might return CinemaHall instance.

Reply:
@ykopatko is right fix ur type annotation